bin/pycbc_submit_dax

```diff
@@ -518,6 +518,12 @@ fi
 WORKFLOW_ID_STRING=""
 WORKFLOW_DB_CMD="sqlite3 -csv ${HOME}/.pegasus/workflow.db \"select submit_hostname,wf_id,wf_uuid from master_workflow where submit_dir = '${SUBMIT_DIR}/work';\""
 DB_TRY=0
+
+# force a condor reschedule to get the workflow running
+set +e
+condor_reschdule &> /dev/null
```

Comment:
Typo?

Reply:
Yes, fixed. You can guess why I didn't spot it...